sjb/config/test_cases/test_pull_request_origin_aggregated_logging_ansible.yml

```diff
@@ -17,7 +17,7 @@ extensions:
       title: "build an openshift-ansible release"
       repository: "openshift-ansible"
       script: |-
-        git checkout "${PULL_REFS%%:*}"
```

Comment:
We should do a check here similar to the other `if` statements for evaluating if its master/es5.x ....